remote-api/src/main/java/org/alfresco/rest/api/model/rules/RuleBuilder.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * #%L
+ * Alfresco Remote API
+ * %%
+ * Copyright (C) 2005 - 2022 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.rest.api.model.rules;
+
+/** Builder class for {@link Rule}. */
+public class RuleBuilder
+{
+    private String id;
+    private String name;
+
+    public RuleBuilder setId(String id)
+    {
+        this.id = id;
+        return this;
+    }
+
+    public RuleBuilder setName(String name)
+    {
+        this.name = name;
+        return this;
+    }
+
+    public Rule createRule()
+    {
+        Rule rule = new Rule();
+        rule.setId(id);
+        rule.setName(name);

```

Comment:
More to follow, I assume ? 

- enabled / disabled ?
- description
- apply to sub-folders (or not)
- other properties (background / foreground, error script for background) ...
- triggers ("when"), conditions, actions, ...

Reply:
Right - these will follow @krdabrowski 's corresponding work on the GET API.